core/modules/catalog/store/category/actions.ts

```diff
@@ -182,7 +183,8 @@ const actions: ActionTree<CategoryState, RootState> = {
   /**
    * Filter category products
    */
-  products (context, { populateAggregations = false, filters = [], searchProductQuery, current = 0, perPage = 50, sort = '', includeFields = null, excludeFields = null, configuration = null, append = false, skipCache = false, cacheOnly = false }) {
+  products (context, { populateAggregations = false, filters = [], searchProductQuery, current = 0, sort = '', includeFields = null, excludeFields = null, configuration = null, append = false, skipCache = false, cacheOnly = false }) {
```

Comment:
as Piotr said, we cannot remove this attribute. You may not have a theme config, or someone would want to fetch some data for other purposes with different size.